### PR TITLE
Update Q1

### PR DESCRIPTION
**Pull request by eliannyscer**
I will add new content

### CODE REVIEW

File: Q1-Difference-between-list-and-tuples.md [hunk 1]
```diff
@@ -3,11 +3,12 @@
...
 * Are mutable they can be edited
-* Are slower than tuples
+* Iteration is slower than tuples
 * You use square bracket in the syntax: list = ["Ellie", 33, 7]
 
 ## TUPLES
 
 * Are immutable (tuples are lists which can’t be edited)
-* Are faster than list
-* You use round brackets in the syntax: tuple = ("Ellie", 33, 7)
```
Comment (mauragas) [inline Q1-Difference-between-list-and-tuples.md]: I think you should add more info

**Review by mauragas — CHANGES_REQUESTED** [review]

File: Q1-Difference-between-list-and-tuples.md (hunk 1 above)
Comment (eliannyscer) [reply Q1-Difference-between-list-and-tuples.md]: I will do it tomorrow!

File: Q1-Difference-between-list-and-tuples.md [hunk 2]
```diff
@@ -2,12 +2,42 @@
 
 ## LIST
 
-* Are mutable they can be edited
-* Are slower than tuples
-* You use square bracket in the syntax: list = ["Ellie", 33, 7]
+`Type` - Are mutable they can be edited, is possible update or delete an item from the list.
```
Comment (mauragas) [inline Q1-Difference-between-list-and-tuples.md]:
```suggestion
`Type` - Are mutable (can be edited), it is possible update or delete an item from the list.
```

File: Q1-Difference-between-list-and-tuples.md [hunk 3]
```diff
@@ -2,12 +2,42 @@
...
+`Syntax` - You use square bracket.
+
+`Example:`
+
+```python3
+supermarket_list = ["Cereal", "bread", "avocado", "beef"]
+print(supermarket_list)
+
+#Output:
+['Cereal', 'bread', 'avocado', 'beef']
```
Comment (mauragas) [inline Q1-Difference-between-list-and-tuples.md]:
```suggestion
# Output:
# ['Cereal', 'bread', 'avocado', 'beef']
```

File: Q1-Difference-between-list-and-tuples.md [hunk 4]
```diff
@@ -2,12 +2,42 @@
...
+`Syntax` - You use round brackets.
+
+`Example:`
+  
+```python3
+tuple = ("Cereal", "bread", "avocado", "beef")
+print(tuple)
+
+#Output:
+('Cereal', 'bread', 'avocado', 'beef')
```
Comment (mauragas) [inline Q1-Difference-between-list-and-tuples.md]:
```suggestion
# Output:
# ('Cereal', 'bread', 'avocado', 'beef')
```

**Review by mauragas — CHANGES_REQUESTED** [review]

**Pull request merged**